### PR TITLE
systemd: Disable atd & cron

### PR DESCRIPTION
**Pull request by KellerFuchs**
Those services can be used to run programs outside a user's slice, bypassing resources limits.

### CODE REVIEW

File: .etckeeper [hunk 1]
```diff
@@ -1,43 +1,50 @@
 # Generated by etckeeper.  Do not edit.
 
-mkdir -p './apache2/mods-available'
-mkdir -p './binfmt.d'
+mkdir -p './apache2/mods-available/mods-available'
```
Comment (daurnimator) [inline .etckeeper]: All these folders have their last component duplicated.

Comment (KellerFuchs) [inline .etckeeper]: Sounds like I created pretty amazing breakage  :V

**Comment by daurnimator** [comment]
@KellerFuchs this PR still has the weird duplication in file names.

**Comment by KellerFuchs** [comment]
@daurnimator Thanks for the reminder.

Also, I just realised that this will break all the maintainance scripts under `cron.*`.
I will add some systemd units that run them.

**Comment by KellerFuchs** [comment]
@daurnimator I added some systemd units to periodically run `/etc/cron.*`, but I didn't try them yet.

**Comment by daurnimator** [comment]
> @daurnimator I added some systemd units to periodically run /etc/cron.*, but I didn't try them yet.

I'd prefer we convert out crontab scripts to systemd timers.
I guess I'm okay with this as a temporary fix though if you want to merge as-is.

**Pull request merged**